core/src/main/java/org/elasticsearch/index/engine/InternalEngine.java

```diff
@@ -1336,10 +1321,26 @@ private void commitIndexWriter(IndexWriter writer, Translog translog, String syn
             }
 
             if (logger.isTraceEnabled()) {
-                logger.trace("committing writer with commit data [{}]", commitData);
+                logger.trace("committing writer with commit data (max_seq_no excluded) [{}]", commitData);
```

Comment:
will do, in doing this, I noticed there is an issue in how the commit data is set - anytime the iterator() is called, it will recompute the maxSeqNo based on the current value of what the SequenceNumbersService returns.  We only want this done once so each subsequent time we call `writer.getLiveCommitData()`, we get the same value that went into the commit.  I'm fixing this and will push up a new commit